augur/merge.py

```diff
@@ -354,6 +379,57 @@ def merge_metadata(args):
             print_info(f"WARNING: Skipped deletion of {db_path} due to error, but you may want to clean it up yourself (e.g. if it's large).")
 
 
+def merge_sequences(args):
+    if not args.skip_input_sequences_validation:
+        for s in args.sequences:
+            print_info(f"Validating {s!r}…")
+
+            with NamedTemporaryFile() as dup_num_file:
```

Comment:
Perhaps

```suggestion
            with NamedTemporaryFile("w+", encoding = "utf-8") as dup_num_file:
```

to make properties clearer from the start? and then the `.decode()` after `.read()` below is also not necessary.

Reply:
Done:

https://github.com/nextstrain/augur/blob/e34a62bdc43a1e12cd8b58277f750a27bb2e20e7/augur/merge.py#L390